docs/community/working-group-definitions.md

```diff
@@ -0,0 +1,195 @@
+* AssemblyLoadContext
+
+It's worth noting that the Engine WG is not responsible for the definition of the PowerShell language.
+This should be handled by the Language WG instead.
+However, it's expected that many issues will require input from both WGs.
+
+### Members
+
+* @daxian-dbw
+* @rjmholt
+* @BrucePay
+
+## Interactive UX
+
+While much of PowerShell can be used through both interactive and non-interactive means,
+some of the PowerShell user experience is exclusively interactive.
+These topics include (but are not limited to):
+
+* Console
+* Help System
+* Tab completion / IntelliSense
+* Markdown rendering
+* PSReadLine
+* Debugging
+
+### Members
+
+* @daxian-dbw (PSReadline / Intellisense)
+* @adityapatwardhan (Markdown / help system)
+* @JamesWTruher (cmdlet design)
+
+## Language
+
+The Language WG is distinct from the Engine WG in that they deal with the abstract definition
+of the PowerShell language itself.
+While all WGs will be working closely with the PowerShell Committee (and may share members),
+it's likely that the Language WG will work especially close with them,
+particularly given the long-lasting effects of language decisions.
+
+### Members
+
+* @JamesWTruher
+* @rjmholt
+* @daxian-dbw
+* @BrucePay
+
+## Remoting
+
+The Remoting WG should focus on topics like the **PowerShell Remoting Protocol (PSRP)**,
+the **protocols implemented under PSRP** (e.g. WinRM and SSH),
+and **other protocols used for remoting** (e.g. "pure SSH" as opposed to SSH over PSRP).
+Given the commonality of serialization boundaries, the Remoting WG should also focus on
+**the PowerShell job system**.
+
+### Members
+
+* @PaulHigin
+* @SteveL-MSFT
+* @TravisEz13
+* @anmenaga
+
+## Modules
+
+For those core/inbox modules whose source code lives within the `PowerShell/PowerShell` repository,
+we should have some set of WGs responsible for their cmdlet design guidelines and enforcement,
+as well as to manage the proposal of new cmdlets or breaking changes.
+
+However, some modules that ship as part of the PowerShell package are managed in other source repositories.
+These modules should be owned by the maintainers of those individual repositories.
+These modules include:
+
+* [`Microsoft.PowerShell.Archive`](https://github.com/PowerShell/Microsoft.PowerShell.Archive)
+* [`PackageManagement` (formerly `OneGet`)](https://github.com/OneGet/oneget)
+* [`PowerShellGet`](https://github.com/PowerShell/PowerShellGet)
+* [`PSDesiredStateConfiguration`](https://github.com/PowerShell/xPSDesiredStateConfiguration)
+  (Note: this community repository maintains a slightly different version of this module on the Gallery,
+  but should be used for future development of `PSDesiredStateConfiguration`.)
+* [`PSReadLine`](https://github.com/PowerShell/PSReadLine)
+* [`ThreadJob`](https://github.com/PowerShell/Modules/tree/master/Modules/Microsoft.PowerShell.ThreadJob)
+
+In the long-term, our desire is to manage all modules using the "separate repo" approach,
```

Comment:
I have never understood this. The only thing you might think is that MSFT PowerShell team has an internal KPI for the number of supported repositories. :-)
The team always says that it is too small and has limited resources, but each additional repository requires additional resources. Even much larger projects (dotnet) prefer a minimum of repositories.

Reply:
Our biggest reasoning is that we ultimately want those modules to be decoupled such that they can deployed to older supported versions of PS7. Similarly, we may want to create distributions of PowerShell in the future that don't contain certain modules.

This is certainly all *possible* within a single repo, but there are tradeoffs. 

Regardless, I'm deferring that conversation as out-of-scope for the creation of WGs.

Reply:
We tried to mitigate the additional cost of maintaining many module repos with the single `module` repo, but that proved untenable for the practical matter that CI/CD couldn't be done on a per module basis vs the entire repo as well as additional cost of triaging issues to mark areas for appropriate experts to review.

Reply:
I have never put pressure on the team in view of the special status of the project, but here I see some confusion.
> Regardless, I'm deferring that conversation as out-of-scope for the creation of WGs.

If we are talking about reducing overhead costs, then it is strongly related areas.
Creating new repositories obviously increases the overhead. Let's look at the .Net team again. They are 10 times larger, but put all the code in one repository. And they publish dozens of _independent_ packages. They use a bot to assign tags. 
Can't we do the same?
We already have _some (10?) labels for modules_ including DSC, PowerShellGet and PSReadline. Add a few more is this really a problem?
Today only one person is working on the vault module. The same for PSReadline module. Don't you want working groups for them? More likely to make them real in a large repository.

> we ultimately want those modules to be decoupled 

This does not contradict in any way placing them in one repository. See again the experience of the .Net team. The maintenance and publishing process, policies, conventions, CIs are the _same_ for all modules. This will significantly reduce overhead costs.

Based on my experience, I would advise MSFT team to go this way if there are no internal restrictions.

Reply:
Updated language to remove future intent